src/containers/Recommendations.js

```diff
@@ -3,21 +3,28 @@ import RecommendationDetails from "../components/RecommendationDetails";
 
 class Recommendations extends Component {
     render() {
+        let sectionName
         if (this.props.resumeBasicInfo) {
-            var sectionName = this.props.resumeBasicInfo.section_name.recommendations;
+          sectionName = this.props.resumeBasicInfo.section_name.recommendations;
         }
+
+        let recos
         if (this.props.resumeRecommendations) {
-          var recoTitle = this.props.resumeRecommendations.title;
-          var recoText = this.props.resumeRecommendations.text;
-          var recoImage = this.props.resumeRecommendations.image;
+          recos = this.props.resumeRecommendations.map((reco, i) => {
+            const title = reco.title;
+            const text = reco.text;
+            const image = reco.image;
+            return <RecommendationDetails key={i} title={title} text={text} image={image} />
+          })
         }
+
         return (
             <section id="recommendations">
                 <div className="col-md-12">
                     <h1 className="section-title" style={{ color: "black" }}>
                         <span>{sectionName}</span>
```

Comment:
and `recos` too?

Reply:
Maybe? xD Are you trying to tell me something?

Reply:
Yeah, a reviewer might ask you something like that because most of the time you need to handle null cases when you work on something more dynamic but in this case:

you actually own the whole thing and it's not gonna change soon, so, it's probably fine to keep it like it ;)